testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py

```diff
@@ -328,6 +328,10 @@ def test_copy_prebuilt_database(params_from_base_test_setup, encrypted):
         prebuilt_db_path = "/assets/{}.cblite2.zip".format(db_prefix)
     elif liteserv_platform == "xamarin-android":
         prebuilt_db_path = "{}.cblite2.zip".format(db_prefix)
+    elif liteserv_platform in ["java-macosx", "java-msft", "java-ubuntu", "java-centos"]:
+        prebuilt_db_path = "{}.cblite2.zip".format(db_prefix)
```

Comment:
line #332 and #334 looks same as line 330. you may need to rearrange , you can put "net-msft" condition and "ios" condition before and rest all in else condition with the logic which is in in line #332